h1st/model_repository/model_repository.py

```diff
@@ -80,17 +80,17 @@ def serialize(self, model, path):
         """
         meta_info = {}
 
-        if hasattr(model, 'metrics'):
+        if model.metrics:
             logger.info('Saving metrics property...')
             meta_info['metrics'] = self.METRICS_PATH
```

Comment:
Why are model metrics being persisted separately from model stats? Are they that decoupled?